assets/js/blocks/active-filters/edit.tsx

```diff
@@ -89,21 +86,23 @@ const Edit = ( {
 	};
 
 	return (
-		<div { ...blockProps }>
```

Comment:
@tjcafferkey I think we should keep the `blockProps` inside the main Edit component of the block instead of exposing that data to the HOC.

Reply:
By doing so, we can update other filters block with minimal changes.

Reply:
In [37910c0](https://github.com/woocommerce/woocommerce-blocks/pull/6907/commits/37910c094b91100c4586d892a576e0268655c7e3) I move the block props back to the edit component, feel free to revert it if it doesn't make sense to you.

Reply:
Makes sense to me @dinhtungdu thanks